### PR TITLE
Address AScan Release Errors/Exceptions

### PR DESCRIPTION
**Pull request by kingthorin**
Per https://twitter.com/psiinon/status/741229375140581376 &
zapbot/zap-mgmt-scripts@205f4cc

TestPathTraversal - catch InvalidRedirectLocationException and
URIException.
TestRemoteFileInclude - adjust logging (debug not error).

### CODE REVIEW

File: src/org/zaproxy/zap/extension/ascanrules/TestPathTraversal.java [hunk 1]
```diff
@@ -457,7 +458,7 @@ private boolean sendAndCheckPayload(String param, String newValue, Pattern patte
         // send the modified request, and see what we get back
 		try {
             sendAndReceive(msg);
-		} catch (SocketException|IllegalStateException|UnknownHostException|IllegalArgumentException ex) {
+		} catch (SocketException|IllegalStateException|UnknownHostException|IllegalArgumentException|InvalidRedirectLocationException ex) {
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/ascanrules/TestPathTraversal.java]: Should also catch URIException? That's another one logged.

File: src/org/zaproxy/zap/extension/ascanrules/TestRemoteFileInclude.java [hunk 2]
```diff
@@ -247,7 +247,10 @@ public void scan(HttpMessage msg, String param, String value) {
             }
 
         } catch (Exception e) {
-            log.error("Error scanning parameters for Path Traversal: " + e.getMessage() + " Caused by: " + e.getCause());
+        	if (log.isDebugEnabled()) {
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/ascanrules/TestRemoteFileInclude.java]: With the log level change, should the exception type being caught changed to a more specific type? Like IOException?

**Comment by thc202** [comment]
The ZapAddOn.xml needs to be updated (bump version and update of "changes").

**Comment by thc202** [comment]
btw, the commit message says "TestPathTraversal adjust logging, also catch..." but the only changes are the catch, should it have other changes?

File: src/org/zaproxy/zap/extension/ascanrules/TestPathTraversal.java (hunk 1 above)
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/ascanrules/TestPathTraversal.java]: One is a result of the other, should we catch both?

File: src/org/zaproxy/zap/extension/ascanrules/TestRemoteFileInclude.java (hunk 2 above)
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/ascanrules/TestRemoteFileInclude.java]: Due to the previous logging, I don't know what the actual exception is (and I don't have a test case for it), it had been logging as a failure caused by null....

**Comment by kingthorin** [comment]
I worded it that way because I also changed the message being logged when caught...

**Comment by thc202** [comment]
Looks good to me. Thanks!

**Pull request merged**